dso/util.py

```diff
@@ -1,7 +1,55 @@
 import functools
 import logging
+import uuid
+
+import dso.model
+import dso.compliancedb.db
+import dso.compliancedb.model
+
+
+logger = logging.getLogger(__name__)
 
 
 @functools.lru_cache
 def component_logger(name: str):
     return logging.getLogger(name)
+
+
+def scans_to_compliance_db(
+    scan_data,
+    tool: dso.compliancedb.model.ScanTool,
+    component_name: str,
+    component_version: str,
+    artifact_name: str,
+    artifact_version: str,
+    artifact_type: dso.model.ArtifactType,
+    compliancedb_cfg_name: str,
+):
+    if not compliancedb_cfg_name:
+        logger.warning('no compliance db cfg name found, skipping insertion')
```

Comment:
as also remarked before: check this at the caller. pass-in the cfg-element, or even better the initialised client here. if there is no compliance-db-cfg, do not create a client and do not call this function. otherwise, call the function, and regard it as an error if no usable client is passed-in